packages/p2p-v2/src/components/Modals/AvailableP2PBalanceModal/AvailableP2PBalanceModal.scss

```diff
@@ -0,0 +1,33 @@
+.p2p-v2-available-p2p-balance-modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    margin-right: -50%;
```

Comment:
do you need this margin-right? cause i can see you're also translating the modal below

Reply:
actually nahh, it works without `margin-right` 😅